### PR TITLE
Added rds-aurora to terraform

### PR DESCRIPTION
**Pull request by cdcampbell24**

### CODE REVIEW

File: terraform/application/modules/rds/rds-aurora.tf [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  type            = "CNAME"
+  zone_id         = var.app_env.hosted_zone_id
+}
+
+resource "aws_db_parameter_group" "sbagov_db_parameter" {
+  description = "mint sbagov db parameter group"
+  family      = "aurora5.6"
+  name        = local.env.parameter_group_name
+  tags = {
+    "Environment" = "mint"
```
Comment (bushong1) [inline terraform/application/modules/rds/rds-aurora.tf:110]: Hardcoded

File: terraform/application/IMPORT_GUIDE.md [hunk 2]
```diff
@@ -116,3 +116,16 @@ terraform import 'module.vpc.aws_network_acl.private[0]' "$(aws ec2 describe-net
 
 ```
 
+# RDS 
+
+``` shell
+
+tf import 'module.rds.module.aurora.aws_rds_cluster.this' $(aws rds describe-db-clusters --query "DBClusters[?contains(DBClusterArn,'$(terraform workspace show)-rds-aurora-sbago')].DBClusterIdentifier" --output text)
+
+tf import 'module.rds.aws_route53_record.sbagov_db_record' "ZFA32P2PPHOAL_content-db.$(terraform workspace show).ussba.io_CNAME"
```
Comment (jsmigovsky) [inline terraform/application/IMPORT_GUIDE.md:125]: split this out for staging and production

File: terraform/application/modules/rds/rds-aurora.tf [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    production = {
+      db_cluster_name         = "production-rds-aurora-sbago-sbagovdatabasecluster-zmhpzel593kn"
+      db_subnet_group_name    = "production-rds-aurora-sbagov-sbagovdatabasesubnetgroup-9cw9eallb60"
+      parameter_group_name    = "production-rds-aurora-sbagov-sbagovdbparametergroup-1120od5w7mm23"
+      backup_retention_period = 35
+    }
+  }
+  env = merge(
+    local.all.default,
+    local.all[terraform.workspace],
```
Comment (jsmigovsky) [inline terraform/application/modules/rds/rds-aurora.tf:42]: lets see if the `key` exists and only use it when it does

File: terraform/application/modules/rds/rds-aurora.tf [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+  version = "~> 2.0"
+  name    = local.env.db_cluster_name
+  engine  = "aurora"
+  # TODO upgrade database version
+  engine_version                  = "5.6.10a"
+  subnets                         = var.private_subnets
+  vpc_id                          = var.vpc_id
+  replica_count                   = 1
+  instance_type                   = "db.t2.medium"
+  apply_immediately               = true
```
Comment (jsmigovsky) [inline terraform/application/modules/rds/rds-aurora.tf:69]: set to `false` or make variable depend on environment

File: terraform/application/modules/rds/rds-aurora.tf [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+  }
+}
+
+module "aurora" {
+  source  = "terraform-aws-modules/rds-aurora/aws"
+  version = "~> 2.0"
+  name    = local.env.db_cluster_name
+  engine  = "aurora"
+  # TODO upgrade database version
+  engine_version                  = "5.6.10a"
```
Comment (jsmigovsky) [inline terraform/application/modules/rds/rds-aurora.tf:64]: make engine version a variable with a default of 5.6.10a

File: terraform/application/modules/rds/rds-aurora.tf [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+  db_subnet_group_name            = aws_db_subnet_group.aurora.name
+  username                        = "sbagov"
+  password                        = data.aws_ssm_parameter.db_password.value
+  backup_retention_period         = local.env.backup_retention_period
+  enabled_cloudwatch_logs_exports = []
+  preferred_backup_window         = "05:00-05:30"
+  preferred_maintenance_window    = "mon:06:00-mon:06:30"
+  db_parameter_group_name         = aws_db_parameter_group.sbagov_db_parameter.id
+  db_cluster_parameter_group_name = "default.aurora5.6"
+  allowed_cidr_blocks             = ["10.20.0.0/20", "20.20.0.0/20"]
```
Comment (jsmigovsky) [inline terraform/application/modules/rds/rds-aurora.tf:80]: delete this, we are passing in a security group

File: terraform/application/rds.tf [hunk 7]
```diff
@@ -0,0 +1,8 @@
+module "rds" {
+  source = "./modules/rds"
+
+  app_env            = local.env
+  vpc_id             = local.vpc_id
```
Comment (jsmigovsky) [inline terraform/application/rds.tf:5]: module.vpc.vpc_id

File: terraform/application/rds.tf [hunk 8]
```diff
@@ -0,0 +1,8 @@
...
+  private_subnets    = local.private_subnets
```
Comment (jsmigovsky) [inline terraform/application/rds.tf:6]: module.vpc.priviate_subnets

**Review by jsmigovsky — CHANGES_REQUESTED** [review]

**Review by bushong1 — APPROVED** [review]

**Review by jsmigovsky — APPROVED** [review]

**Pull request merged**